BrainPortal/lib/generators/cbrain_task/templates/show_params.html.erb

```diff
@@ -12,7 +12,7 @@
 #
 #    task  -  Your current task object
 #
-#    params -  A shortend to task.params
+#    params -  A shortened to task.params
```

Comment:
This was changed from 'shortend' to 'shortened' but the real word was supposed to be `shorthand`. Also happens in the next file in this PR.